README.md

```diff
@@ -551,4 +552,48 @@ _Jonathan Hersh, Allen Langmaier_
 - **Updates to development**
   - Dynamic updates to list elements
   - `CPListItem`
-  - `CPListImageRowItem`
\ No newline at end of file
+  - `CPListImageRowItem`
+
+## Streamline your app clip
+
+https://developer.apple.com/wwdc20/10120/
+
+_Yongjun Zhang, Luming Yin_
+
+- **Best practices**
+  - Focus on essential tasks
+  - Limit the features. Reserve complex features for your app
+  - Require all assets for initial experience
+  - Do not include splash screens
+  - Avoid requiring people to sign up _before_ finishing their task
+  - Only ask for location and notification permissions if needed (more on this below)
+  - Make sure your App Clip has the same name and icon as your app
+  - Put shared assets in a shared asset catalog
+  - Authentication
+    - Support sign in with apple
```

Comment:
Never thought I will ever corect a typo for you 😅 But here it is:

**Sign in with Apple**

Reply:
hahaha 😆 good catch!

Reply:
```suggestion
    - Support Sign in with Apple
```